### PR TITLE
Add getWiki endpoint

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/WikiService.Business/Commands/Wiki/GetWikiFilter.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.WikiService.Business.Commands.Wiki
+{
+  public record GetWikiFilter : IRequest<List<RubricData>>
+  {
+    [FromQuery(Name = "includearchivals")]
+    public bool IncludeArchivals { get; set; } = false;
```
Comment (ndoriomedov) [inline src/WikiService.Business/Commands/Wiki/GetWikiFilter.cs]: IncludeArchived maybe?

File: src/WikiService.Business/Commands/Wiki/GetWikiHandler.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          Name = x.Name,
+          IsActive = x.IsActive,
+          ParentId = x.ParentId,
+          ArticlesNames = x.Articles.Select(article => article.Name).ToList()
+        }).ToListAsync();
+
+        if (!request.IncludeArchivals)
+        {
+          rubrics.RemoveAll(x => !x.IsActive);
+        }
```
Comment (ndoriomedov) [inline src/WikiService.Business/Commands/Wiki/GetWikiHandler.cs]: maybe do it before materialization?

File: src/WikiService/Controllers/WikiTreeController.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      _mediator = mediator;
+    }
+
+    [HttpGet("get")]
+    public async Task<IActionResult> GetAsync(
+      [FromQuery] GetWikiFilter request,
+      CancellationToken ct)
+    {
+      return Ok(await _mediator.Send(request, ct));
```
Comment (ndoriomedov) [inline src/WikiService/Controllers/WikiTreeController.cs:27]: can it be not 200?

**Review by ndoriomedov — APPROVED** [review]

File: Common.Configurations.targets [hunk 4]
```diff
@@ -26,10 +26,12 @@
   </PropertyGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.62" />
+    <PackageReference Include="MediatR" Version="10.0.1" />
+    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="10.0.1" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.63" />
     <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.6" />
```
Comment (MariyaSin) [inline Common.Configurations.targets]: could you update the package?

File: src/WikiService.Business/Commands/Wiki/GetWikiFilter.cs (hunk 1 above)
Comment (MariyaSin) [reply src/WikiService.Business/Commands/Wiki/GetWikiFilter.cs]: maybe includeDeactivated?) like in other services)

File: src/WikiService.Business/Commands/Wiki/GetWikiHandler.cs (hunk 2 above)
Comment (MariyaSin) [reply src/WikiService.Business/Commands/Wiki/GetWikiHandler.cs]: could you add Where condition?

File: src/WikiService/Controllers/WikiTreeController.cs (hunk 3 above)
Comment (MariyaSin) [reply src/WikiService/Controllers/WikiTreeController.cs:27]: there is no way to get other statusCode

File: Common.Configurations.targets (hunk 4 above)
Comment (VLiamin) [reply Common.Configurations.targets]: done

File: src/WikiService.Business/Commands/Wiki/GetWikiHandler.cs (hunk 2 above)
Comment (VLiamin) [reply src/WikiService.Business/Commands/Wiki/GetWikiHandler.cs]: done

File: src/WikiService.Business/Commands/Wiki/GetWikiFilter.cs (hunk 1 above)
Comment (VLiamin) [reply src/WikiService.Business/Commands/Wiki/GetWikiFilter.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**